### PR TITLE
Feature/playground

### PR DESCRIPTION
**Pull request by jezemery**
- Add new team members and remove old
- Add team link to header to improve potential comms
- Add next meetup date to home page
- Add a nicer donate button

### CODE REVIEW

File: templates/app/index.phtml [hunk 1]
```diff
@@ -13,6 +13,7 @@ $this->headTitle('Welcome!');
 
         <h1>Our meetups...</h1>
         <p>Our meetups take place on every <strong>2<sup>nd</sup> Wednesday of the month from 7pm at the <a href="http://goo.gl/maps/ET78d">Oasis Conference Centre (Arundel Street)</a> in Portsmouth</strong>.</p>
+        <p>The next meetup will be happening on <?php echo $this->nextMeet ?> at 7pm</p>
```
Comment (asgrim) [inline templates/app/index.phtml]: What if the meetup isn't happening (as happened last month)? I think this data should be sourced from the meetups data...

File: templates/app/team.phtml [hunk 2]
```diff
@@ -10,53 +10,51 @@ $this->headTitle($title);
...
     <ul id="team">
         <li>
-            <img src="/images/team/jez.jpg" />
+            <img src="/images/team/jez.jpg"/>
             <h2>Jez Emery</h2>
             <p>
-                Jez works at Strawberrysoup working with their clients and infrastructure. Jez has been an active member of the PHP
+                Jez works at Strawberrysoup working with their clients and infrastructure. Jez has been an active member
+                of the PHP
                 Hampshire community for a few years now, and his relentless enthusiasm for building the community has
```
Comment (asgrim) [inline templates/app/team.phtml]: Sort alignment?

File: public-src/phphants/scss/style.scss [hunk 3]
```diff
@@ -499,18 +503,23 @@ ul#videos .video {
     }
 }
 
-ul#team li {
+ul#team li, ul#past-leaders li {
```
Comment (asgrim) [inline public-src/phphants/scss/style.scss]: This css addition implies new markup added, but I don't see it? Either add past leaders or drop this unused definition :)

File: public-src/phphants/scss/style.scss [hunk 4]
```diff
@@ -499,18 +503,23 @@ ul#videos .video {
...
     overflow: hidden; /* clearfix */
     padding-bottom: 20px;
     list-style-type: none;
 }
-ul#team li img {
+
+ul#team li img, ul#past-leaders li img {
```
Comment (asgrim) [inline public-src/phphants/scss/style.scss]: Same

File: public-src/phphants/scss/style.scss [hunk 5]
```diff
@@ -499,18 +503,23 @@ ul#videos .video {
...
     width: 220px;
     height: 200px;
     float: left;
     padding-right: 20px;
 }
 
+ul#past-leaders li img {
```
Comment (asgrim) [inline public-src/phphants/scss/style.scss]: Same

File: src/App/Action/IndexAction.php [hunk 6]
```diff
@@ -23,6 +23,19 @@ public function __construct(TemplateRendererInterface $templateRenderer)
 
     public function __invoke(Request $request, Response $response, callable $next = null) : HtmlResponse
     {
-        return new HtmlResponse($this->templateRenderer->render('app::index'));
+        $currentDate = strtotime(date('U'));
```
Comment (asgrim) [inline src/App/Action/IndexAction.php]: Lots of terrible assumptions and undefined behaviour when doing time maths like this. Use `\DateTimeImmutable` with `\DateTimeZone` (in UTC) wherever possible, and convert to the client timezone for display purposes only.

File: src/App/Action/IndexAction.php [hunk 7]
```diff
@@ -23,6 +23,19 @@ public function __construct(TemplateRendererInterface $templateRenderer)
...
+        $nextMeet    = strtotime('second wednesday of '.date('M'));
+        if ($currentDate >= $nextMeet) {
+            $nextMeet = strtotime('second wednesday of '.date('M', strtotime("+1 month")));
```
Comment (asgrim) [inline src/App/Action/IndexAction.php]: What happens on Jan 31st?

File: src/App/Action/IndexAction.php [hunk 8]
```diff
@@ -23,6 +23,19 @@ public function __construct(TemplateRendererInterface $templateRenderer)
...
+        }
+
+        return new HtmlResponse(
+            $this->templateRenderer->render(
+                'app::index',
+                [
+                    'nextMeet' => date('l jS \of F Y', $nextMeet),
```
Comment (asgrim) [inline src/App/Action/IndexAction.php]: Need some test cases to ensure this doesn't break, or switch to using the meetup service (or make a new `interface`).

File: templates/app/team.phtml (hunk 2 above)
Comment (jezemery) [reply templates/app/team.phtml]: eh?

Comment (asgrim) [reply templates/app/team.phtml]: PHP / Hampshire is split across lines unnecessarily, just a cosmetic thing!

**Comment by jezemery** [comment]
@asgrim you happy with this now?

File: public/js/scripts.min.js [hunk 9]
```diff
@@ -1 +1 @@
...
...meborder="0" allowfullscreen></iframe>').addClass("lazyYT-video-loaded")}),t.getJSON("//gdata.youtube.com/feeds/api/videos/"+u+"?v=2&alt=json",function(t){a.find("#lazyYT-title-"+u).text(t.entry.title.$t)})}t.fn.lazyYT=function(e){var l={loading_text:"Loading...",default_ratio:"16:9",callback:null,container_class:"lazyYT-container"},n=t.extend(l,e);return this.each(function(){a(t(this).addClass(n.container_class),n)})}}(jQuery),$(document).ready(function(){$(".lazyload-youtube").lazyYT(),$(".nav-button").click(function(){$("nav").toggleClass("nav--shown"),$("nav").hasClass("nav--shown")?$(".nav-button").text("Close"):$(".nav-button").text("Menu")}),$(".ask-confirmation").click(function(t){t.preventDefault(),confirm("Are you sure you wish to perform this action?")&&(window.location.href=$(this).attr("href"))}),$(".attending-button").click(function(t){var a=$(this);t.preventDefault(),$.post(a.attr("href")).done(function(t){if(t.attending)return a.addClass("is-attending"),void a.text(t.isPast?"I went to this":"I will be there");a.removeClass("is-attending"),a.text(t.isPast?"I wasn't there":"I won't be there")}).fail(function(){alert("Could not toggle your attendance status...")})})});
```
Comment (lboynton) [inline public/js/scripts.min.js]: Why has this changed?

Comment (asgrim) [reply public/js/scripts.min.js]: @jezemery @lboynton maybe something didn't get changed before? Jez could you try generating this again just to be sure please? If it's really changed that's ok :+1:

Comment (lboynton) [reply public/js/scripts.min.js]: Looking at the diff the two versions are logically the same, so must be minifier difference..? Probably fine anyway.

Comment (jezemery) [reply public/js/scripts.min.js]: It was the minifier, I assumed there was perhaps some legacy versioning or something that changed

**Pull request merged**